docs/2/technical/data-format/overview.md

```diff
@@ -114,55 +107,49 @@ Here we will see one by one the different components, starting with the unique v
 "updaterId": "yourname@microshare.io"
 ```
 
-* `"_id"`
+* `"_id"` and `"id"`
 
 The Id of the data is its unique identifier. It is used by a Microshare® Technician to find a particular piece of data in the datalake.
 
 * `"checksum"`
 
-The checksum is another identifier also for other needs for Microshare®.
+The checksum is another identifier that Microshare® technicians use to find that peice of data in the datalake.
```

Comment:
The checksum is a cryptographic hash of the data portion of the record. Using the sha-256 hashing algorithm to generate a new hash from the contents of the Data element, a comparison can be made to ensure that the data has not be compromised. A checksum is also included in the Origin section of the JSON structure that captures the data contents when the record was first introduced to the system. By comparing these two checksums, you can prove that the data has not be modified inside the Microshare system.